web/src/containers/ActivityDetailContractorResources.js

```diff
@@ -135,27 +158,46 @@ class ContractorExpenses extends Component {
                   <Label>
                     {t('activities.contractorResources.labels.attachments')}
                   </Label>
-                  <div className="md-col-9 md-flex">
-                    <div className="md-col-4">
+                  <div className="md-col-9 md-flex items-start">
+                    <div className="md-col-6 flex items-end mr2">
+                      <Select
+                        name={`${contractor.key}-attachment-type`}
+                        options={DOC_TYPES}
+                        label="Document Type"
+                        wrapperClass="col-6 mr2"
+                        value={docType}
+                        onChange={this.updateDocType}
+                      />
                       <Dropzone
                         className="btn btn-primary"
-                        onDrop={this.handleDrop(i)}
+                        onDrop={this.handleFileUpload(i)}
                       >
```

Comment:
Should we add a `multiple={false}` prop to `Dropzone` to match the handler behavior?

Reply:
YES!!